price-estimator/src/orderbook.rs

```diff
@@ -63,12 +70,15 @@ impl Orderbook {
         }
     }
 
-    /// Recreate the pricegraph orderbook.
+    /// Recreate the pricegraph orderbook and update the infallible price source.
```

Comment:
nit: we could add a unit test that the infallible price source is actually updated on `orderbook.upate`

Reply:
Added test